tests/test_algebra_complex.py

```diff
@@ -21,8 +19,7 @@ class TestAlgebraComplex(unittest.TestCase):
 
     @unittest.skipIf(Complex64TensorType is None,
                      reason="not available")
-    @unittest.skipIf(StrictVersion(onnx.__version__) < StrictVersion('1.8.0'),
-                     reason="not implemented")
+    @unittest.skipIf(TARGET_OPSET < 13, reason="not implemented")
```

Comment:
Similar to above should it be  ` <14`？

Reply:
See https://github.com/onnx/onnx/blob/main/docs/Versioning.md.